app/controllers/index.rb

```diff
@@ -0,0 +1,14 @@
+enable :sessions
```

Comment:
`enable :sessions` should be in an `app` or `config` file not one of you controllers